### PR TITLE
Expand pool registration documentation

### PR DESCRIPTION
**Pull request by xdzurman**
Describes pool registration and refers to original documentation.

Should solve #19

### CODE REVIEW

File: docs/poolRegistration.md [hunk 1]
```diff
@@ -158,7 +221,9 @@ sudo cardano-hw-cli shelley transaction witness \
 
 ## Create signed transaction 
 
- ```
+Use witnesses from previous step to sign the pool registration transaction
```
Comment (refi93) [inline docs/poolRegistration.md:224]: "to assemble the signed pool registration transaction" would be more accurate as the witnesses are already signatures in their own right

**Review by refi93 — APPROVED** [review]
one small remark, otherwise LGTM

File: docs/poolRegistration.md [hunk 2]
```diff
@@ -124,31 +187,31 @@ cardano-cli shelley transaction build-raw \
...
+--fee 194685 \
 --out-file tx.raw \
 --certificate-file pool-registration.cert
 ```
 
-## Sign the transaction
+## Create transaction witnesses
 
-This includes creating witnesses with the operator, pool and for all included pool owners
+This includes creating witnesses with the operator, pool and for all included pool owners. Only pool owners can provide their witness from a hardware wallet.
```
Comment (refi93) [inline docs/poolRegistration.md:197]: Maybe explicitly clarify that the other keys (cold keys/spending keys) need to be managed in the cli.

**Comment by xdzurman** [comment]
Squash merged through console

**Pull request closed without merging**